pkg/reconciler/broker/broker.go

```diff
@@ -235,18 +241,26 @@ func (r *Reconciler) reconcile(ctx context.Context, b *v1alpha1.Broker) error {
 	})
 
 	ingressChannelName := resources.BrokerChannelName(b.Name, "ingress")
-	ingressChan, err := r.reconcileIngressChannel(ctx, ingressChannelName, channelResourceInterface, b)
+	ingressChannelObjRef := corev1.ObjectReference{
+		Kind:       b.Spec.ChannelTemplate.Kind,
+		APIVersion: b.Spec.ChannelTemplate.APIVersion,
+		Name:       ingressChannelName,
+		Namespace:  b.Namespace,
+	}
+
+	// Start tracking the ingress channel.
+	if err = track(ingressChannelObjRef); err != nil {
+		logging.FromContext(ctx).Error("Unable to track changes to Channel", zap.Error(err))
```

Comment:
```suggestion
		logging.FromContext(ctx).Error("Unable to track changes to Channel", zap.Error(err), zap.Any("channel", ingressChannelObjRef))
```

Reply:
Done, thanks!

Reply:
ended up annotating the error and returning, as in other places